src/utilities/core/CommonInclude.i

```diff
@@ -66,7 +66,7 @@
 // and nested classes and "nothing known about <type>"
 // also ignoring #509, shadowed method. Its a red herring for all of our cases
 //#pragma SWIG nowarn=401,362,365,366,368,378,503,801,312
-#pragma SWIG nowarn=362,365,366,368,378,503,801,312,509,401
+#pragma SWIG nowarn=362,365,366,368,378,503,801,312,509,401,516
```

Comment:
Avoid thousands of "overloaded method ignore, using XXX instead". In particular for every modelObject, the Ctor is overloaded (rule of 5/6) so it was getting out of control.